lib/ansiblelint/rules/ModuleTemplateExt.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016, Tsukinowa Inc. <info@tsukinowa.jp>
```

Comment:
I'm heavily against this because syntax highlighting tends to be based on the extension, and so you don't get proper syntax highlighting for .j2 files

Please remove this and move it to an optional rule.

Reply:
This was removed